cmd/neofs-cli/modules/netmap.go

```diff
@@ -19,13 +28,79 @@ var netmapCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(netmapCmd)
 
-	// Here you will define your flags and configuration settings.
+	netmapCmd.AddCommand(
+		getEpochCmd,
+		localNodeInfoCmd,
+	)
+
+	localNodeInfoCmd.Flags().BoolVar(&nodeInfoJSON, "json", false, "print node info in JSON format")
+}
+
+var getEpochCmd = &cobra.Command{
+	Use:   "epoch",
+	Short: "Get current epoch number",
+	Long:  "Get current epoch number",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cli, err := getSDKClient()
+		if err != nil {
+			return err
+		}
+
+		e, err := cli.Epoch(context.Background())
+		if err != nil {
+			return fmt.Errorf("rpc error: %w", err)
+		}
+
+		fmt.Println(e)
+
+		return nil
+	},
+}
+
+var localNodeInfoCmd = &cobra.Command{
+	Use:   "nodeinfo",
+	Short: "Get local node info",
+	Long:  `Get local node info`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cli, err := getSDKClient()
+		if err != nil {
+			return err
+		}
+
+		nodeInfo, err := cli.EndpointInfo(context.Background())
+		if err != nil {
+			return fmt.Errorf("rpc error: %w", err)
+		}
+
+		prettyPrintNodeInfo(nodeInfo, nodeInfoJSON)
+
+		return nil
+	},
+}
+
+func prettyPrintNodeInfo(i *netmap.NodeInfo, jsonEncoding bool) {
+	if jsonEncoding {
+		data, err := netmap.NodeInfoToJSON(i)
+		if err != nil {
+			printVerbose("Can't convert container to json: %w", err)
+			return
+		}
+
+		buf := new(bytes.Buffer)
+		if err := json.Indent(buf, data, "", "  "); err != nil {
+			printVerbose("Can't pretty print json: %w", err)
+		}
+
+		fmt.Println(buf)
+
+		return
+	}
 
-	// Cobra supports Persistent Flags which will work for this command
-	// and all subcommands, e.g.:
-	// netmapCmd.PersistentFlags().String("foo", "", "A help for foo")
+	fmt.Println("key:", base58.Encode(i.PublicKey()))
```

Comment:
I would like to encode it in hex. It is only 33 byte long and we already using hex encoding for hashes, checksums and verbose output of generated private key. We use base58 mostly for identifiers such as object ID, container ID, owner ID.

Reply:
Fixed with extra commit.